client/src/containers/frontend/ProjectResources.js

```diff
@@ -109,6 +128,7 @@ class ProjectResourcesContainer extends Component {
             project={project}
             resources={this.props.resources}
             pagination={this.props.meta.pagination}
+            paginationClickHandler={this.pageChangeHandlerCreator}
```

Comment:
You're passing a function in a prop that's supposed to be an object, per your proptype validation above.